### PR TITLE
add GA4GH FASP GWAS demo workflow container specs

### PR DESCRIPTION
**Pull request by uniqueg**
The PR includes a `Dockerfile` and scripts required for building a container image used for the demonstration of the ELIXIR Cloud & AAI stack at the Federated Analysis Systems Project (FASP) session of the 2020 Global Alliance for Genomic and Health (GA4GH) Plenary, **showcasing the integration of various ELIXIR GA4GH API implementations with the Biocontainers TRS API**, as discussed with @ypriverol

The ELIXIR Cloud & AAI team would be grateful for merging this as soon as integration of the core ELIXIR Cloud stack is a long overdue but ultimately last minute addition, with the demo happening in a week's time.**

 # Submitting a Container

 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [ ] LABEL about.documentation   
  - [ ] LABEL about.license_file
  - [ ] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

File: gwas-fasp/v1.90p/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Base image
+FROM ubuntu:bionic
+
+# Metadata
+LABEL base_image="ubuntu:bionic"
+LABEL version="1"
+LABEL software="PLINK"
```
Comment (osallou) [inline gwas-fasp/v1.90p/Dockerfile]: Software label name should match directory name, gaws-fasp in your case

File: gwas-fasp/v1.90p/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+LABEL software.version="v1.9p"
+LABEL about.summary="Container image for GWAS analysis workflow for GA4GH Plenary 2020 FASP demonstrator containging PLINK and custom scripts"
+LABEL about.home="https://www.cog-genomics.org/plink2"
+LABEL about.license="SPDX:GPL-3.0-or-later"
+LABEL extra.identifiers.biotools=comet
```
Comment (osallou) [inline gwas-fasp/v1.90p/Dockerfile]: Please add about.license_file with url to license

**Review by osallou — CHANGES_REQUESTED** [review]

File: gwas-fasp/v1.90p/Dockerfile (hunk 1 above)
Comment (uniqueg) [reply gwas-fasp/v1.90p/Dockerfile]: Thanks - makes sense!

**Review by osallou — APPROVED** [review]

**Comment by osallou** [comment]
@uniqueg merged

**Pull request merged**